### ISSUE TITLE
Fullscreen button should be disabled for frameless window on OS X 10.9

### ISSUE THREAD

**Issue by miniak** [electron/electron#6227]
* Electron version: 1.2.5
* Operating system: OS X 10.9.5

This seems to be related to the `titlebarStylingToolbar`, which is being created in `windowDidEnterFullScreen`, which is not needed on OS X 10.9 as it does not support the titleBarStyle.

**Comment by zcbenz** [comment]
As commented in https://github.com/electron/electron/pull/6228#issuecomment-228500043, this is probably a bug in your own fork.

**Comment by miniak** [comment]
@zcbenz This is happening in the unmodified prebuilt version. Do you have access to OS X 10.9 to check it out? Or I can take a screencast and post the video demonstrating the issue.

**Comment by zcbenz** [comment]
Sorry I was mislead by the comment, after looking into it, I found this behavior was actually used to fix a bug: https://github.com/electron/electron/issues/88, I didn't realize that user may actually want to have the fullscreen button.

**Comment by miniak** [comment]
The question is also whether a frameless window should be fullscreensble by default.

**Comment by zcbenz** [comment]
The behavior of frameless window used to be: it is fullscreenable but the fullscreen button is always hidden. But after some version we broke the behavior and it became: it is fullscreenable and the fullscreen button is visible.

I don't know when the behavior was changed, apparently only few people tested frameless window on OS X 10.9. I think we should just restore old behavior by always disabling the fullscreen button for frameless window on OS X 10.9.
